kube/init/starboard-cm.yaml

```diff
@@ -0,0 +1,193 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app.kubernetes.io/managed-by: starboard
+  name: starboard
+  namespace: starboard
+data:
```

Comment:
This is now a generic ConfigMap. We can use it to as Volume Mount of type ConfigMap or ConfigMapKeyRef in scan Job descriptors. We can also access the values programmatically with the client-go library.